packages/fxa-settings/src/pages/CannotCreateAccount/index.tsx

```diff
@@ -6,21 +6,37 @@ import React from 'react';
 import LinkExternal from 'fxa-react/components/LinkExternal';
 import { RouteComponentProps } from '@reach/router';
 import AppLayout from '../../components/AppLayout';
+import CardHeader from '../../components/CardHeader';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { logPageViewEvent } from '../../lib/metrics';
 
-// NOTE: page is incomplete, was half-hacked together as a first route to serve
+export const routeName = 'cannot_create_account';
 
 const CannotCreateAccount = (_: RouteComponentProps) => {
+  /* TODO: get serviceName from relier once FXA-6437 is complete. Or... rethink this issue
+   * filed/fixed 8.5 years ago: https://github.com/mozilla/fxa-content-server/issues/1783
+   * In some contexts, opening this link in a new tab was not ideal, so we previously set it
+   * to open in a new tab _only_ if it was the Sync flow. Is this still true?
```

Comment:
Alternatively... maybe we _always_ want this to open in the same tab. If users are already locked out, all they can do is close the tab on this page.

Reply:
It looks like `about:accounts` is no longer a thing, so I'm not sure what that is now. For Sync context, it just means going through the sign up flow through Sync, which is when we set `target` to `blank`. I'm not sure what "marketplace-dev" is though, I'd asked Vijay and he said it might've been test pilot? Anyways I'm just going to file a follow up issue about this to determine what we want here now.